tests/view_tests.py

```diff
@@ -1,4 +1,4 @@
-#coding: utf-8
+# -*- coding: utf-8 -*-
```

Comment:
If this change is going to be made it should probably be made to all the files in the project rather than a one off basis. Personally I don't think it really matters, so I'm fine with no change too.